src/main/java/org/prebid/server/spring/config/bidder/AdnuntiusBidderConfiguration.java

```diff
@@ -24,20 +28,39 @@ public class AdnuntiusBidderConfiguration {
 
     @Bean("adnuntiusConfigurationProperties")
     @ConfigurationProperties("adapters.adnuntius")
-    BidderConfigurationProperties configurationProperties() {
-        return new BidderConfigurationProperties();
+    AdnuntiusConfigurationProperties configurationProperties() {
+        return new AdnuntiusConfigurationProperties();
     }
 
     @Bean
-    BidderDeps adnuntiusBidderDeps(BidderConfigurationProperties adnuntiusConfigurationProperties,
+    BidderDeps adnuntiusBidderDeps(AdnuntiusConfigurationProperties adnuntiusConfigurationProperties,
                                    @NotBlank @Value("${external-url}") String externalUrl,
                                    Clock clock,
                                    JacksonMapper mapper) {
 
-        return BidderDepsAssembler.forBidder(BIDDER_NAME)
+        return BidderDepsAssembler.<AdnuntiusConfigurationProperties>forBidder(BIDDER_NAME)
                 .withConfig(adnuntiusConfigurationProperties)
                 .usersyncerCreator(UsersyncerCreator.create(externalUrl))
-                .bidderCreator(config -> new AdnuntiusBidder(config.getEndpoint(), clock, mapper))
+                .bidderCreator(config -> new AdnuntiusBidder(
+                        config.getEndpoint(),
+                        ObjectUtil.getIfNotNull(config.getExtraInfo(), ExtraInfo::getUrl),
+                        clock,
+                        mapper))
                 .assemble();
     }
+
+    @Data
+    @EqualsAndHashCode(callSuper = true)
+    @NoArgsConstructor
+    private static class AdnuntiusConfigurationProperties extends BidderConfigurationProperties {
+
+        private ExtraInfo extraInfo = new ExtraInfo();
+    }
+
+    @Data
+    @NoArgsConstructor
+    private static class ExtraInfo {
+
+        String url;
+    }
```

Comment:
`GO` uses `ExtraInfa` because that's the only way to customize their configuration, we can have them as simple properties without the need for this "wrapper". What do you think guys @And1sS @Net-burst ?
```
     @Data
     @EqualsAndHashCode(callSuper = true)
     @NoArgsConstructor
     private static class AdnuntiusConfigurationProperties extends BidderConfigurationProperties {

         private String euEndpoint
     }
```